app/views/content/salaries-and-benefits.md

```diff
@@ -56,57 +56,52 @@ keywords:
 
 $new-salaries$
 
-For helping to shape the next generation, you’re entitled to a competitive salary, generous holidays, and a substantial pension.
+If you’re a primary or secondary teacher in England, your salary will depend on the type of school you work in, where, and the pay range you’re on. 
 
-These salaries apply to maintained primary, secondary and special schools in England (schools funded by local authorities).
+These pay ranges apply to maintained schools (schools funded by local authorities). 
 
-Other types of schools (for example, academies) can set their own pay scales.
+Non-maintained schools (for example, academies, independent schools and private schools) can set their own pay. 
 
-Figures apply from September 2023.
+Figures apply from 1 September 2023.
 
-## Primary and secondary teaching salaries
+## Primary and secondary teacher salary
 
-All qualified primary and secondary teachers will have a starting salary of at least £30,000. This will be higher for teachers working in London. 
+You’ll start on at least £30,000 (or more in London) as a primary or secondary school teacher in England. 
 
-### Qualified teacher salary
+Your salary will be reviewed every year, with most teachers moving up the pay range annually (this will depend on your school’s performance management arrangements).    
 
-Your school will have their own pay scales for qualified teachers. Pay increases will always be linked to performance, not length of service, and will be reviewed every year.
+A typical teacher with 5 years’ experience will earn at least £41,333 (or more in London).
 
-The teacher pay scales for qualified teachers are:
+### Qualified teacher salary
```

Comment:
@plisovin is there a way we can make this into a caption for the below table, like how it describes in the [WCAG guidelines?](https://www.w3.org/WAI/tutorials/tables/caption-summary/)

Reply:
@EllieNodder @plisovin We can't have captions on the site yet unfortunately as there is an iOS/VoiceOver bug which renders them meaningless (see https://trello.com/c/mjikZkhz) for iPhone/VoiceOver users - VoiceOver reads out a long string of numbers as part of the caption

Reply:
:( sadness